### PR TITLE
[zh-cn]: create the translation of the MathML global attribute `dir`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20746

### CODE REVIEW

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### 可选值
+
+- `ltr`，表示 _从左到右_，用于将数学表达式从左到右渲染（例如，英语或摩洛哥风格）；
+- `rtl`，表示 _从右到左_，用于将数学表达式从右到左渲染（例如，摩洛哥或马赫雷克风格）；
+
+> **备注：**
+>
+> - 如果存在 CSS 页面并且元素支持这些属性，则此属性可以被 CSS 属性 {{ cssxref("direction") }} 覆盖。
+> - 由于数学的方向性与其内容而不是其呈现方式在语义上相关，因此建议 Web 开发人员在可能的情况下使用此属性而不是相关的 CSS 属性。这样，即使在不支持 CSS 或将 CSS 禁用的浏览器上，公式也将正确显示。
+> - `dir` 属性用于设置数学公式的方向性，在阿拉伯语世界通常是从右到左的。然而，从右到左书写的语言通常嵌入从左到右书写的数学内容。因此，默认情况下，HTML `dir` 属性中的 auto 关键字不被识别，并且[用户代理样式表](/zh-CN/docs/Web/CSS/Cascade#user-agent_stylesheets)会重置 [math](/zh-CN/docs/Web/MathML/Element/math) 元素上的方向属性。
```
Comment (T34-active) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:49]:
```suggestion
> - `dir` 属性用于设置数学公式的方向性，在阿拉伯语世界通常是从右到左的。然而，从右到左书写的语言通常嵌入从左到右书写的数学内容。因此，默认情况下，HTML `dir` 属性中的 auto 关键字不被识别，并且[用户代理样式表](/zh-CN/docs/Web/CSS/Cascade#用户代理样式表)会重置 [math](/zh-CN/docs/Web/MathML/Element/math) 元素上的方向属性。
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: dir
+slug: Web/MathML/Global_attributes/dir
+l10n:
+  sourceCommit: 8eece0b998c23e8ea35f936d7371a169974130f5
+---
+
+{{MathMLRef}}
+
+**`dir`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)是一个[枚举](/zh-CN/docs/Glossary/Enumerated)属性，表示 MathML 元素的方向性。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:10]:
```suggestion
**`dir`** [全局属性](/zh-CN/docs/Web/MathML/Global_attributes)是一个表示 MathML 元素的方向的[枚举](/zh-CN/docs/Glossary/Enumerated)属性。
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## 语法
+
+```html
+<!-- Moroccan style -->
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:15]:
```suggestion
<!-- 摩洛哥风格 -->
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    <mi>س</mi>
+  </msqrt>
+  <mo>=</mo>
+  <msup>
+    <mn>3</mn>
+    <mi>ب</mi>
+  </msup>
+</math>
+
+<!-- Maghreb/Machrek style -->
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:27]:
```suggestion
<!-- 马格里布/马什里克风格 -->
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  </msqrt>
+  <mo>=</mo>
+  <msup>
+    <mn>٣</mn>
+    <mi>ب</mi>
+  </msup>
+</math>
+```
+
+### 可选值
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:40]:
```suggestion
### 值
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+- `ltr`，表示 _从左到右_，用于将数学表达式从左到右渲染（例如，英语或摩洛哥风格）；
+- `rtl`，表示 _从右到左_，用于将数学表达式从右到左渲染（例如，摩洛哥或马赫雷克风格）；
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:43]:
```suggestion
- `ltr`，表示*从左到右*，用于将数学表达式从左到右渲染（例如，英语或摩洛哥风格）；
- `rtl`，表示*从右到左*，用于将数学表达式从右到左渲染（例如，摩洛哥或马什里克风格）；
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+> **备注：**
+>
+> - 如果存在 CSS 页面并且元素支持这些属性，则此属性可以被 CSS 属性 {{ cssxref("direction") }} 覆盖。
+> - 由于数学的方向性与其内容而不是其呈现方式在语义上相关，因此建议 Web 开发人员在可能的情况下使用此属性而不是相关的 CSS 属性。这样，即使在不支持 CSS 或将 CSS 禁用的浏览器上，公式也将正确显示。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:48]: “由于数学的方向性与其内容而不是其呈现方式在语义上相关”语法有错误

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+> - `dir` 属性用于设置数学公式的方向性，在阿拉伯语世界通常是从右到左的。然而，从右到左书写的语言通常嵌入从左到右书写的数学内容。因此，默认情况下，HTML `dir` 属性中的 auto 关键字不被识别，并且[用户代理样式表](/zh-CN/docs/Web/CSS/Cascade#用户代理样式表)会重置 [math](/zh-CN/docs/Web/MathML/Element/math) 元素上的方向属性。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:49]:
```suggestion
> - `dir` 属性用于设置数学公式的方向性，在阿拉伯语世界通常是从右到左的。然而，从右到左书写的语言通常嵌入从左到右书写的数学内容。因此，默认情况下，HTML `dir` 属性中的 `auto` 关键字不被识别，并且[用户代理样式表](/zh-CN/docs/Web/CSS/Cascade#用户代理样式表)会重置 [`math`](/zh-CN/docs/Web/MathML/Element/math) 元素上的方向属性。
```

File: files/zh-cn/web/mathml/global_attributes/dir/index.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+### 值
+
+- `ltr`，表示*从左到右*，用于将数学表达式从左到右渲染（例如，英语或摩洛哥风格）；
+- `rtl`，表示*从右到左*，用于将数学表达式从右到左渲染（例如，摩洛哥或马什里克风格）；
+
+> **备注：**
+>
+> - 如果存在 CSS 页面并且元素支持这些属性，则此属性可以被 CSS 属性 {{ cssxref("direction") }} 覆盖。
+> - 数学的方向性与其内容相关，而不是与其表现形式相关，因此建议网络开发者尽可能使用这一属性，而不是相关的 CSS 属性。这样，即使在不支持 CSS 或已停用 CSS 的浏览器上，公式也能正确显示。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/global_attributes/dir/index.md:48]:
```suggestion
> - 数学的方向性与其内容的语义相关，而不是与其表现形式相关，因此建议 Web 开发者尽可能使用这一属性，而不是相关的 CSS 属性。这样，即使在不支持 CSS 或已停用 CSS 的浏览器上，公式也能正确显示。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**